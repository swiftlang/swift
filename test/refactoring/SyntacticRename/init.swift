struct Test {
    var base: Int
    /*test:def*/init(base: Int) {}
}

_ = /*test:call*/Test(base: 3)
_ = Test . /*test:call*/init(base: 3)
_ = Test . /*test:ref*/init
_ = Test . /*test:ref*/init(base:)


// REQUIRES: swift_swift_parser
// RUN: %empty-directory(%t.ranges)
// RUN: %refactor -find-rename-ranges -source-filename %s -pos="test" -is-function-like -old-name "init(base:)" >> %t.ranges/init.swift.expected
// RUN: diff -u %S/Outputs/init.swift.expected %t.ranges/init.swift.expected

