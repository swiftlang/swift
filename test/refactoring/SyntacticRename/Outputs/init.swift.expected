struct Test {
    var base: Int
    /*test:def*/<keywordBase>init</keywordBase>(<arglabel index=0>base</arglabel><param index=0></param>: Int) {}
}

_ = /*test:call*/Test(<callarg index=0>base</callarg><callcolon index=0>: </callcolon>3)
_ = Test . /*test:call*/<keywordBase>init</keywordBase>(<callarg index=0>base</callarg><callcolon index=0>: </callcolon>3)
_ = Test . /*test:ref*/<keywordBase>init</keywordBase>
_ = Test . /*test:ref*/<keywordBase>init</keywordBase>(<sel index=0>base</sel>:)



